integrations/access/common/app.go

```diff
@@ -111,7 +111,7 @@ func (a *BaseApp) initTeleport(ctx context.Context, conf PluginConfiguration) (c
 		return "", "", trace.Wrap(err)
 	}
 
-	if pong.ServerFeatures.AdvancedAccessWorkflows {
+	if pong.ServerFeatures.AccessRequests.Enabled {
```

Comment:
This sets off some very bright red flags for me, we need to carefully consider backwards compatibility. Integrations on slightly older versions than their auth server will be checking the `AdvancedAccessWorkflows` flag.

Reply:
yes 👍